### ISSUE TITLE
IE 8 compatibility issue

### ISSUE THREAD

**Issue by bluecode2** [Mottie/tablesorter#1431]
I'm sorry if this is a stupid question, but I'm really stuck now.
I already develop all my needs in chrome, but my users (like another users out there) still using IE8 for their primary browser. The problem is, when I try to open it using IE8, it pops up an error like this.

Webpage error details

User Agent: Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 6.1; WOW64; Trident/4.0; SLCC2; .NET CLR 2.0.50727; .NET CLR 3.5.30729; .NET CLR 3.0.30729; Media Center PC 6.0; InfoPath.3; .NET4.0C; .NET4.0E)
Timestamp: Wed, 19 Jul 2017 10:20:06 UTC

Message: **Expected identifier, string or number**
Line: 1
Char: 2697
Code: 0
URI: https://10.20.200.7/script/jquery.tablesorter.min.js

I thought there's something wrong with my code, but when I try to make a new blank page which is only including jquery.tablesorter.min.js without using it, the same error occurs. Do you have any idea about this error?

**Comment by Mottie** [comment]
Hi @bluecode2!

Please share the code you are using and use the un-minified version of the plugin to get a more reasonable line number error to work with. Also, what version of jQuery is being used?

**Comment by bluecode2** [comment]
wowww,, I tried to use the un-minified version and it works!! Seems like I'll use the un-minified one for the time being. Thank you for your advice.
Anyway, I because of this strange solution, I tried to compare both minified and un-minified version. I found the difference in this part of code (line 204)

// digit sort, text location
		string : {
			max      : 1,
			min      : -1,
			emptymin : 1,
			emptymax : -1,
			zero     : 0,
			none     : 0,
			**'null'   : 0,**
			top      : true,
			bottom   : false
		},

in minified version, you write **null** (without quotes) instead of **'null'**. I don't really know how IE 8 interpret it, but that's the only thing that I found for now. Once again, thank you very much.
